### PR TITLE
fix: exclusiveMin/Max shows incorect range

### PR DESCRIPTION
**Pull request by Oprysk**
## What/Why/How?
fix: https://github.com/Redocly/redoc/issues/1761
## Check yourself

- [x] Code is linted
- [x] Tested
- [x] All new/updated code is covered with tests

### CODE REVIEW

File: src/utils/openapi.ts [hunk 1]
```diff
@@ -448,16 +452,18 @@ export function humanizeConstraints(schema: OpenAPISchema): string[] {
...
-    let maximum = 0;
-    if (schema.minimum) minimum = schema.minimum;
-    if (typeof schema.exclusiveMinimum === 'number') minimum = minimum <= schema.exclusiveMinimum ? minimum : schema.exclusiveMinimum;
-
-    if (schema.maximum) maximum = schema.maximum;
-    if (typeof schema.exclusiveMaximum === 'number') maximum = maximum > schema.exclusiveMaximum ? maximum : schema.exclusiveMaximum;
-
-    numberRange = `[${minimum} .. ${maximum}]`
+  if (typeof schema.exclusiveMinimum === 'number' && typeof schema.exclusiveMaximum === 'number') {
+    numberRange = '[';
```
Comment (RomanHotsiy) [inline src/utils/openapi.ts:456]: Exclusive minimum means bigger, not bigger or equal. In math notation this corresponds to `(` vs `[`.
```suggestion
    numberRange = '(';
```

**Review by RomanHotsiy — CHANGES_REQUESTED** [review]
This still has some issue:

```
exclusiveMinimum: 5
maximum: 10
```

Won't work correctly with this code.

Should show:

```
(5 .. 10]
```

File: src/utils/openapi.ts (hunk 1 above)
Comment (Oprysk) [reply src/utils/openapi.ts:456]: thanks!
I've already cover this case

File: src/utils/openapi.ts [hunk 2]
```diff
@@ -415,6 +419,29 @@ function humanizeRangeConstraint(
...
 }
 
+export function humanizeNumberRange(schema: OpenAPISchema): string | undefined {
+  const minimum =
+    typeof schema.exclusiveMinimum === 'number'
+      ? Math.min(schema.exclusiveMinimum, schema.minimum ?? Infinity)
+      : schema.minimum;
+  const maximum =
+    typeof schema.exclusiveMaximum === 'number'
+      ? Math.max(schema.exclusiveMaximum, schema.maximum ?? 0)
```
Comment (RomanHotsiy) [inline src/utils/openapi.ts:429]: It won't work for negative numbers. Would be good to add a test for this case maybe.
```suggestion
      ? Math.max(schema.exclusiveMaximum, schema.maximum ?? -Infinity)
```

**Review by RomanHotsiy — CHANGES_REQUESTED** [review]
Left one small comment.

File: src/utils/openapi.ts (hunk 2 above)
Comment (Oprysk) [reply src/utils/openapi.ts:429]: good point, thanks!

**Review by RomanHotsiy — APPROVED** [review]
🔥

**Review by AlexVarchuk — APPROVED** [review]

**Pull request merged**